server/controllers/ssotRetrievalController.js

```diff
@@ -16,34 +16,34 @@ exports.getSingleSourceOfTruth = async (req, res) => {
     }
 };
 
-// GET /getAvailableBotsForUser/:userid
+// GET /getAvailableBotsForUser/78d09f66d2ed466cf20b06f7
 exports.getBotList = async (req, res) => {
     try {
         res.set('Content-Type', 'application/json');
         const { userid } = req.params;
-        
         const usableUserId = mongoose.Types.ObjectId(userid);
-        const availableSSOTsById = await mongoose.model('userAccessObject').find(
-            { userId: usableUserId }, 
+
+        const userAccessObjs = await mongoose.model('userAccessObject').find(
+            { userId: usableUserId },
             {
                 AccessLevel: 0,
                 _id: 0,
                 userId: 0
             }
         ).exec();
-        
+
         const SSOTIds = [];
-        availableSSOTsById.forEach((singleUserObj) => {
+        userAccessObjs.forEach((singleUserObj) => {
             SSOTIds.push(singleUserObj.robotId);
         });
 
         const availableSSOTs = await mongoose.model('SSoT').find(
-            { _id: { $in: SSOTIds } }, 
+            { _id: { $in: SSOTIds } },
             {
                 robotMetadata: 1
             }
         ).exec();
-        
+
         const SSOTs = [];
         availableSSOTs.forEach((singleSSOT) => {
             SSOTs.push(singleSSOT.robotMetadata);
```

Comment:
I would rather use camelCase consistently and change SSOTIds to ssotIds

Reply:
fixed